### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by MTahir001**
This is a full animation project, let me know if any modifications required

### CODE REVIEW

**Comment by SaadMahi** [comment]
make it mobile responsive.

**Review by nehalskdev — APPROVED** [review]
LGTM nice project just try to make it more responsive on a smaller screen

**Review by ManSOorcode — APPROVED** [review]
good

File: style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  min-height: 100vh;
+  margin: 0;
+
+  font-family: "Source Code Pro", monospace;
+  text-align: center;
+}
+
+h1 {
+  position: relative;
+  font-size: clamp(1rem, 3vw + 1rem, 4rem);
```
Comment (murtazshaikh) [inline style.css:24]: What does clamp do ?

**Review by murtazshaikh — APPROVED** [review]
Keep it up

**Review by anask601 — APPROVED** [review]
Great

File: style.css (hunk 1 above)
Comment (MTahir001) [reply style.css:24]: By using clamp, we can give a max and min limit to our font-size. 
Or you can say clamp give constraint to our font-size

**Comment by MTahir001** [comment]
> make it mobile responsive.

i did it, pls check

**Review by SaadMahi — COMMENTED** [review]
LGTM

File: style.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+body {
+  display: grid;
+  place-items: center;
+
+  background: var(--bg-color);
+  min-height: 100vh;
+  margin: 0;
+  min-width: 380px;
```
Comment (Abbas4tech) [inline style.css:16]: Use relative units

**Review by Abbas4tech — APPROVED** [review]
Minor changes required else LGTM

File: style.css (hunk 2 above)
Comment (MTahir001) [reply style.css:16]: oh sure

**Pull request merged**